01-Authorization-RS256/main.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	auth0 "github.com/auth0-community/go-auth0"
+	"github.com/gorilla/mux"
+	jose "gopkg.in/square/go-jose.v2"
+	jwt "gopkg.in/square/go-jose.v2/jwt"
+)
+
+const JWKS_URI = "https://{YOUR-AUTH0-DOMAIN}.auth0.com/.well-known/jwks.json"
```

Comment:
the values for these variables should be populated by the sample packager which means we'll need to use `{DOMAIN}` and `{API_IDENTIFIER}`